README.md

```diff
@@ -1,2 +1,155 @@
+
+```java
+IGraphServiceClient graphClient =
+  GraphServiceClient
+    .builder()
+    .authenticationProvider(msalAuthenticationProvider)
+    .buildClient();
+```
+
+## 3. Make requests against the service
+
+After you have a GraphServiceClient object, you can begin making calls against the service. The requests against the service look like our [REST API](https://developer.microsoft.com/en-us/graph/docs/concepts/overview).
+
+### 3.1 Get the user's drive
+
+To retrieve the user's drive:
+
+```java
+graphClient
+  .me()
+  .drive()
+  .buildRequest()
+  .get(new ICallback<Drive>() {
+     @Override
+     public void success(final Drive result) {
+        System.out.println("Found Drive " + result.id);
+     }
+     ...
+     // Handle failure case
+  });
+```
+
+## 4. Documentation
+
+For more detailed documentation, see:
+
+* [Overview](https://github.com/microsoftgraph/msgraph-sdk-java/wiki/Overview)
+* [Extending the library](https://github.com/microsoftgraph/msgraph-sdk-java/wiki/Extending-the-Library)
+* [Handling Open Types, PATCH support with `null` values](https://github.com/microsoftgraph/msgraph-sdk-java/wiki/Working-with-Open-Types)
+* [Collections](https://github.com/microsoftgraph/msgraph-sdk-java/wiki/Working-with-Collections)
+* [Making custom requests](https://github.com/microsoftgraph/msgraph-sdk-java/wiki/Custom-Requests)
+* [Known issues](https://github.com/microsoftgraph/msgraph-sdk-java/wiki/Known-Issues)
+* [Contributions](https://github.com/microsoftgraph/msgraph-sdk-java/blob/master/CONTRIBUTING.md)
+
+## 5. Issues
+
+For known issues, see [issues](https://github.com/microsoftgraph/msgraph-sdk-android-auth/issues).
+
+## 6. Contributions
+
+The Microsoft Graph SDK is open for contribution. To contribute to this project, see [Contributing](https://github.com/microsoftgraph/msgraph-sdk-java/blob/master/CONTRIBUTING.md).
+
+Thanks to everyone who has already devoted time to improving the library:
+
+<!-- ALL-CONTRIBUTORS-LIST:START  -->
+<!-- prettier-ignore -->
+| [<img src="https://avatars3.githubusercontent.com/u/16473684?v=4" width="100px;"/><br /><sub><b>Nakul Sabharwal</b></sub>](https://developer.microsoft.com/graph)<br />[](#question-NakulSabharwal "Answering Questions") [](https://github.com/microsoftgraph/msgraph-sdk-java/commits?author=NakulSabharwal "Code") [](https://github.com/microsoftgraph/msgraph-sdk-java/wiki "Documentation") [](#review-NakulSabharwal "Reviewed Pull Requests") [](https://github.com/microsoftgraph/msgraph-sdk-java/commits?author=NakulSabharwal "Tests")| [<img src="https://avatars2.githubusercontent.com/u/3197588?v=4" width="100px;"/><br /><sub><b>Deepak Agrawal</b></sub>](https://github.com/deepak2016)<br />
```

Comment:
point to msgraph-sdk-android-auth repository

Reply:
changed link to android auth